src/library/banshee/bansheeplaylistmodel.cpp

```diff
@@ -287,7 +287,7 @@ void BansheePlaylistModel::trackLoaded(QString group, TrackPointer pTrack) {
         if (pTrack) {
             for (int row = 0; row < rowCount(); ++row) {
                 QUrl rowUrl(getFieldString(index(row, 0), CLM_URI));
-                if (rowUrl.toLocalFile() == pTrack->getLocation()) {
+                if (TrackFile::fromUrl(rowUrl).location() == pTrack->getLocation()) {
```

Comment:
I can imagine that this way to compare urls is hard to discover. May we can add a equals() function to clarify this.

Reply:
Getting the QUrl from pTrack is more obvious and will clarify this situation. I will check for more places where  TrackFile::fromUrl() is used.

Reply:
Misunderstanding. Of course, operator==() is already available.